Resume-6-12.md

```diff
@@ -0,0 +1,44 @@
+# Resume
+
+## Q6. Memory managed
+
+* The computer memory is like a empty book. Different writers want to write some story there but to do it they need to ask for permission to the book's manager because this show where they can write without over write the other story.
+* When anybody is reference or using the story somebody delete it to have more space for the new ones.
+* `The writer` are the `applications`, `the book's manager` is the `memory manager` and the person that `remove` the story is the `garbage collection`.
+* Additionally you can think in the `reference` like the `number of the page` in the book where you can find the story(address).
+
+## Q7. Name-scapes
+
+This make `immune our programs from name conflicts` because is checking if the names in our program are unique. The name is a unique identifier and the space is the place where you can access to the variable.
```

Comment:
```suggestion
Name-scapes makes `our programs immune from name conflicts` because it is checking if the names in our program are unique. The name is a unique identifier and the space is the place where you can access to the variable.
```